internal/worker/trace/client.go

```diff
@@ -129,10 +138,19 @@ func NewClient(ctx context.Context, namespace coretrace.TaggedTracerNamespace, e
 		return nil, nil, nil, errors.Trace(err)
 	}
 
+	bsp := sdktrace.NewBatchSpanProcessor(exporter,
+		sdktrace.WithMaxExportBatchSize(512),
```

Comment:
Any reasoning behind these magic numbers?

Reply:
Looks like these are both exported and defaults... https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/trace/batch_span_processor.go

Reply:
I was playing around with these, because I want to make these configurable.